packages/restlessness-cli/src/commands/dev/index.ts

```diff
@@ -1,38 +1,94 @@
 import minimist from 'minimist';
 import path from 'path';
 import { spawn } from 'child_process';
+import chalk from 'chalk';
+
+const printProcessData = (procName, color, data) => {
+  process.stdout.write(`${color(procName)}: ${data.toString()}`);
+};
+const printProcessError = (procName, err) => {
+  process.stderr.write(`${chalk.red(procName)}: ${err.toString()}`);
+};
+
+function spawnProject(name) {
+  const proc = spawn('serverless', ['offline', '--port', '4000'], {
+    shell: true,
+  });
+  proc.stdout.on('data', d => {
+    printProcessData(name, chalk.green, d);
+  });
+  proc.stderr.on('data', e => {
+    printProcessError(name, e);
+  });
+  proc.on('close', (code) => {
+    console.log(`Frontend process exited with code ${code}`);
+  });
+  proc.on('error', console.error);
+  return proc;
+}
 
 export default async (argv: minimist.ParsedArgs) => {
   const majorVersion: number = parseInt((/^(\d+)(\.\d+){0,2}$/.exec(process.versions.node))[1], 10);
   if (majorVersion < 12) {
     throw new Error('Run command requires node version >= 12.x');
   }
 
+  let projectName;
+  let projectProc;
+  try {
+    projectName = require(path.join(process.cwd(), 'package.json')).name;
+  } catch {
+    console.log(chalk.red('Cannot find package.json. Are you on the root folder?'));
+    process.exit(1);
+  }
+
   const backend = spawn('serverless', ['offline', '--port', '4123'], {
     cwd: path.join(__dirname, '..', '..', '..', 'lib', 'assets', 'backend'),
     env: {
       ...process.env,
       RLN_PROJECT_PATH: process.cwd(),
     },
-    stdio: 'inherit',
+    stdio: ['pipe', 'pipe', 'pipe', 'ipc'],
     shell: true,
   });
-  backend.on('error', console.log);
+  backend.stdout.on('data', d => {
+    printProcessData('rln-backend', chalk.yellow, d);
```

Comment:
I think "Restlessness" is better than "rln-backed" from a user point of view, to refer to what the framework is doing

Reply:
I agree😄